_posts/2019-12-13-basic-rails-container.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "A Basic Rails Container"
```

Comment:
@rdormer Maybe "How to create a basic Rails container" or something similar sounds better for the title?